index.html

```diff
@@ -1151,7 +1154,98 @@ <h2>Constrainable Properties</h2>
+        the instant the frame capture process began, which is useful for example for
+        delay measurements and synchronization.
+        A source of frames can define how this timestamp is set, otherwise it is unset. A
+        sink of frames can define how this timestamp is used if set.
+      </p>
+      <p>
+        Each frame may have an absolute <dfn class="export">receive timestamp</dfn> representing
+        the last received timestamp of packets used to produce this video frame was received in its entirety.
+        The timestamp is useful for example for network jitter measurements.
+        A source of frames can define how this timestamp is set, otherwise it is unset. A sink of
+        frames can define how this timestamp is used if set.
+      </p>
+      <p>
+        Each frame may have a <dfn class="export">RTP timestamp</dfn> representing the packet RTP
+        timestamp used to produce this video frame. The timestamp is useful for example for frame
+        identification and playback quality measurements. A source of frames can define how the
+        timestamp is set, otherwise it is unset. A sink of frames can define how this timestamp is
+        used if set.
+        The packet RTP timestamp concept is defined in [[?RFC3550]] Section 5.1.
+      </p>
+      <h3>{{VideoFrameMetadata}}</h3>
+        <pre class="idl">
+partial dictionary VideoFrameMetadata {
+  DOMHighResTimeStamp captureTime;
+  DOMHighResTimeStamp receiveTime;
+  unsigned long rtpTimestamp;
+};</pre>
+      <section class="notoc">
+        <h5>Members</h5>
+        <dl class="dictionary-members" data-link-for="VideoFrameMetadata" data-dfn-for="VideoFrameMetadata">
+          <dt><dfn><code>captureTime</code></dfn> of type <span class="idlMemberType">DOMHighResTimeStamp</span></dt>
+          <dd>
+            <p>The capture timestamp of the frame relative to {{Performance}}.{{Performance/timeOrigin}}. It corresponds to
+              the [=capture timestamp=] of {{MediaStreamTrack}} video frames.
+            </p>
+          </dd>
+          <dt><dfn><code>receiveTime</code></dfn> of type <span class="idlMemberType">DOMHighResTimeStamp</span></dt>
+          <dd>
+            <p>The receive time of the corresponding encoded frame relative to {{Performance}}.{{Performance/timeOrigin}}.
+              It corresponds to the [=receive timestamp=] of {{MediaStreamTrack}} video frames.</p>
+          </dd>
+          <dt><dfn><code>rtpTimestamp</code></dfn> of type <span class="idlMemberType">unsigned long</span></dt>
+          <dd>
+            <p>The RTP timestamp of the corresponding encoded frame.  It corresponds to [=RTP timestamp=] of
+            {{MediaStreamTrack}} video frames.</p>
+          </dd>
+        </dl>
+      </section>
+      <h3>Algorithms</h3>
+      The <dfn class="export">Initialize Video Frame Timestamps From Internal MediaStreamTrack Video Frame</dfn>
```

Comment:
```suggestion
      The <dfn class="abstract-op">Initialize Video Frame Timestamps From Internal MediaStreamTrack Video Frame</dfn>
```
(this marks the definition as being the name of an algorithm, and has it exported automatically)